update.sh

```diff
@@ -160,8 +160,7 @@ for version in "${versions[@]}"; do
 
 		# Libraries to build the nis module available in Alpine 3.7, but also require this patch:
 		# https://bugs.python.org/issue32521
-		# TODO: Remove Python version check once 2.7 and 3.6 have the patch
-		if [[ "$variant" == alpine* ]] && [[ "$variant" != alpine3.7 || "$version" != 3.7* ]]; then
+		if [[ "$variant" == alpine* ]] && [[ "$variant" != alpine3.7 ]]; then
```

Comment:
Won't this also leave the libraries on `python:2.7-alpine3.7`?  Is that version fixed too?

Reply:
I did think so when I wrote this the first time, but the Python 2.7 Dockerfiles actually aren’t templated.

Reply:
You are correct; I forgot about these lines https://github.com/docker-library/python/blob/33b16b4747706dd47ade94bebfde6b1144e4b758/update.sh#L135-L136